compiler-lib/src/firm/method_body_generator.rs

```diff
@@ -302,7 +307,11 @@ impl<'a, 'ir, 'src, 'ast> MethodBodyGenerator<'ir, 'src, 'ast> {
             Some(mode) => Value(ValueComputation::simple(
                 &call.project_result_tuple().project(mode, 0),
             )),
-            None => Void,
+            None => {
+                // TODO is this problematic?
```

Comment:
don't know if `this` referrs to the keep_alive, but I guess the keep_alive is problematic at some point